boilerplate/app/components/Toggle.tsx

```diff
@@ -346,15 +376,13 @@ function Switch(props: ToggleInputProps) {
     detailStyle: $detailStyleOverride,
   } = props
 
-  const knob = useRef<Animated.View>()
-  const [renderedKnobWidth, setRenderedKnobWidth] = useState(null)
+  const knobWidth = [$detailStyleOverride?.width, $switchDetail?.width, 24].filter(
```

Comment:
Do we want to assign `24` to something like `defaultSize` for readability?

Reply:
Good idea 👍🏻